### ISSUE TITLE
✨ Admin / Skills editor / mostly Styling update

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#11476]
## ✨ Feature

Updating the skills editor to our new admin patterns

### Out of scope for this issue (don't build this even if it's in the design)
- [ ] Updates to the table component
- [ ] Moving to the new nav
- [ ] Advanced tools tab (and everything in it)
- [ ] Search and filter on the table

## 🎨 Design

[Figma file](https://www.figma.com/design/tsgPpY1zCyvdOLCHstimEP/Admin---Skills?node-id=5437-34104&t=fgGVqReY9F7peFvC-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Skills editor page (/admin/settings/skills)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
- [ ] Create skill (/admin/settings/skills/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] (should use same component as Skill information)
- [ ] New page - Skills information (/admin/skills/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Only one tab for now
  - [ ] Inline form for displaying information and editing
- [ ] Edit a skill page (/admin/skills/:id:/edit)
  - [ ] (should use same component as Create Skill page)

- [ ] Mobile view for all pages

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

**Comment by gobyrne** [comment]
And this one @NienkeBr, no rush... also I might stop tagging you

**Comment by gobyrne** [comment]
Need to confirm "soft delete" is in fact what we want for "archive". But it looks like it isn't.

Probably we should break off the "archive" work into a separate ticket.

**Comment by gobyrne** [comment]
Add filter dialogue design @Jerryescandon  (if it's not already in the Figma)

**Comment by Jerryescandon** [comment]
> Add filter dialogue design @Jerryescandon (if it's not already in the Figma)

Done! 👍
